src/MasterChef.sol

```diff
@@ -245,11 +245,7 @@ contract MasterChef is Ownable2StepUpgradeable, IMasterChef {
      * @return The MOE per second for the pool ID.
      */
     function getMoePerSecondForPid(uint256 pid) external view override returns (uint256) {
```

Comment:
I'm really nitpicking but the comment "Else, it will return the MOE per second multiplied by the proportion of votes" isn't exactly accurate anymore